inc/tag.class.php

```diff
@@ -75,7 +75,17 @@ public static function getBlacklistItemtype()
      */
     public static function canItemtype($itemtype = '')
     {
-        return !empty($itemtype) && class_exists($itemtype) && !in_array($itemtype, self::getBlacklistItemtype());
+        $tags = new self();
+        $types_menu = [];
+        $use_global_tag = false;
+        foreach ($tags->find(['is_active' => 1]) as $tag) {
+            if ($tag['type_menu'] != null) {
+                $types_menu = array_merge($types_menu, json_decode($tag['type_menu']));
+            } else {
+                $use_global_tag = true;
+            }
+        }
+        return !empty($itemtype) && class_exists($itemtype) && !in_array($itemtype, self::getBlacklistItemtype()) && ($use_global_tag == true ? $use_global_tag : in_array($itemtype, $types_menu));
```

Comment:
Just to simplify reading:
```suggestion
        return !empty($itemtype) &&
            class_exists($itemtype) &&
            !in_array($itemtype, self::getBlacklistItemtype()) &&
            ($use_global_tag || in_array($itemtype, $types_menu));
```

Reply:
suggestion 

```php
public static function canItemtype($itemtype = '')
{
    if (empty($itemtype) || !class_exists($itemtype) || in_array($itemtype, self::getBlacklistItemtype())) {
        return false;
    }

    $tags = new self();
    $types_menu = [];
    $use_global_tag = false;

    foreach ($tags->find(['is_active' => 1]) as $tag) {
        if (!empty($tag['type_menu'])) {
            $types_menu = array_merge($types_menu, json_decode($tag['type_menu']));
        } else {
            $use_global_tag = true;
        }
    }

    return $use_global_tag || in_array($itemtype, $types_menu);
}
```